src/unit_tests/os_auth/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ list(APPEND os_auth_flags "${DEBUG_OP_WRAPPERS} \
                            -Wl,--wrap,os_random -Wl,--wrap,GetRandomNoise -Wl,--wrap,getuname -Wl,--wrap,time")
 list(APPEND os_auth_names "test_generate_cert")
 list (APPEND os_auth_flags "-Wl,--wrap,PEM_write_PrivateKey -Wl,--wrap,PEM_write_X509 -Wl,--wrap,RSA_generate_key_ex \
-                            -Wl,--wrap,X509_sign -Wl,--wrap,_merror \
+                            -Wl,--wrap,X509_sign -Wl,--wrap,_merror -Wl,--wrap,popen \
```

Comment:
The wrap for function "popen" is not needed in this test, since it is not used by the called functions in it.

Reply:
Yes, it is necessary because the `__wrap_popen` mock was modified to use test_mode, and it must be added so that it finds `__real_popen`, although it is not used directly, if there are `popen` calls in the test

Reply:
Ditto

Reply:
As discussed in the huddle, it was determined that they were necessary.